lib/rules/jsx-pascal-case.js

```diff
@@ -62,7 +62,12 @@ module.exports = {
         const isPascalCase = PASCAL_CASE_REGEX.test(name);
         const isCompatTag = jsxUtil.isDOMComponent(node);
         const isAllowedAllCaps = allowAllCaps && ALL_CAPS_TAG_REGEX.test(name);
-        const isIgnored = ignore.indexOf(name) !== -1;
+        const isIgnored = ignore.find(stringOrRegexp => {
+          if (stringOrRegexp instanceof RegExp) {
```

Comment:
i don't think rules should be required to have a JS config to make them useful (camelcase rule notwithstanding); so it'd only ever get strings, not regex instances.

Also, `instanceof` is never reliable and should never be used; use https://npmjs.com/is-regex if you need to check for this.